src/scripts/user_script_rhenvar2_cp_fix.lua

```diff
@@ -0,0 +1,29 @@
+--
+-- RhenVar2CPFix by Rayman1103
+-- Description: Fixes issue where Command Posts 1 and 4 aren't counted as valid CPs for Rhen Var Citadel Conquest.
+--
+
+--attempt to take control of (or listen to the calls of) the ScriptPostLoad function
+if ScriptPostLoad and not RhenVar2CPFix_ScriptPostLoad then
+	--backup the current ScriptPostLoad function
+	RhenVar2CPFix_ScriptPostLoad = ScriptPostLoad
```

Comment:
any reason for this not to be local?

Reply:
I guess I never gave it a thought. Is local better in this situation? If so, I can update this and the previous Hero user_scripts and change those calls to local as well.

Reply:
If it is intended that it be acted upon outside this script, then it should be global; otherwise local.

Reply:
Alright, I'll update them later and make them local. Thanks for the advice.